### PR TITLE
Feature/#30

### LINKED ISSUE #30: Height colouring

### PR DESCRIPTION
**Pull request by szymonzygzula**

### CODE REVIEW

File: include/controller/terrain_translator.hpp [hunk 1]
```diff
@@ -6,11 +6,13 @@
 #include <view/opengl_interfacing/vertex.hpp>
 #include <model/terrain.hpp>
 #include <view/opengl_interfacing/mesh.hpp>
+#include "controller/terrain_colorer.hpp"
 
 class TerrainTranslator
 {
 private:
 	Terrain* terrain;
+	TerrainColorer* colorer;
```
Comment (TransientTetra) [inline include/controller/terrain_translator.hpp]: Please change this to a normal variable, not of pointer type

File: src/controller/terrain_translator.cpp [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 TerrainTranslator::TerrainTranslator(Terrain *terrain)
 : terrain(terrain)
-{}
+{
+	colorer = new TerrainColorer();
```
Comment (TransientTetra) [inline src/controller/terrain_translator.cpp]: Please remove this once colorer is changed to normal variable

**Review by TransientTetra — CHANGES_REQUESTED** [review]
Great work, only a minute change to colorer member field

**Comment by szymonzygzula** [comment]
Mistakes have been corrected

**Review by TransientTetra — APPROVED** [review]
All clear, go ahead and merge

**Pull request merged**